source/manual/rename-a-country.html.md

```diff
@@ -130,5 +130,6 @@ If the old country slug is still appearing in the search results, this can be re
 
 1. Navigate to the `Search results` page and enter the path to be removed
 2. Remove result, it should now disappear from the listing.
```

Comment:
Unless a specific reason for keeping the documentation regarding using the Search Admin is around, I'd suggest not mentioning it, and just mentioning the Whitehall approach which you're adding in.

Reply:
I'm happy with that, I'll update it now 👍